packages/boxel/addon/components/boxel/input/token-select/index.css

```
@@ -0,0 +1,43 @@
+.boxel-input-token-select {
+  --boxel-input-token-select-font-size: var(--boxel-font-size);
+  --boxel-input-token-select-border-color: var(--boxel-form-control-border-color);
+  --boxel-input-token-select-border-radius: var(--boxel-form-control-border-radius);
+
+  align-items: center;
+  border-radius: var(--boxel-input-token-select-border-radius);
+  border: 1px solid var(--boxel-input-token-select-border-color);
+  display: flex;
+  font-family: var(--boxel-font-family);
+  font-size: var(--boxel-input-selectable-token-amount-input-font-size);
```

```suggestion
  font-size: var(--boxel-input-token-select-font-size);
```

I found that the font size didn’t change in the preview deployment

---

Thanks. I removed it.